composer.json

```diff
@@ -35,7 +35,8 @@
     "symfony/config": "^2.8|^3.0",
     "symfony/dependency-injection": "^2.8|^3.0",
     "symfony/http-kernel": "^2.8|^3.0",
-    "symfony/yaml": "^2.8|^3.0"
+    "symfony/yaml": "^2.8|^3.0",
+    "symfony/console": "^2.8|^3.0"
```

Comment:
Symfony 3.0/3.1/3.2 are not maintained anymore (http://symfony.com/roadmap), see #77. If we all agree, this one should be bumped to `~3.3` and the PR rebased, tests will be green again.

Reply:
Can be rebased :)